style.css

```diff
@@ -31,14 +31,13 @@
   height: fit-content;
   width: fit-content;
   float: left;
-  margin-bottom: 12px;
 }
 
 .hamburger {
   height: 12px;
   float: right;
   width: fit-content;
-  margin-bottom: 16px;
+  margin-bottom: 29px;
```

Comment:
- [OPTIONAL] Good job so far! Please pay attention a little bit more, kindly decrease the top of your nav bar 😃 

![image](https://user-images.githubusercontent.com/63915024/226941885-f6e0a024-4607-45a4-9c81-e5fbe6b92d08.png)

Reply:
Hey, thank you for making clear what the previous reviewer was trying to communicate. Now I understand what his review meant.